### ISSUE TITLE
Can I use this source file?

### ISSUE THREAD

**Issue by LeePC** [towerdefence-cc/tower-defence#1]
I want to use it for twitch broadcasting
Can I use it?

**Comment by ZakShearman** [comment]
Of course. Bear in mind I wouldn't recommend running this in production yet as it still has bugs and the mechanics aren't yet finished ❤️

I'm going to make a project board on this org specifically for tracking TowerDefence changes that need to be made.

Note that any changes to the source must abide by the LGPL-2.1 license.

**Comment by ZakShearman** [comment]
Board now available here: https://github.com/orgs/towerdefence-cc/projects/1/views/5
